### DISCUSSION TITLE
Can you schedule an update from an update server?

### DISCUSSION THREAD

**Discussion by mdemin914** [Hieromon/AutoConnect#439] [Feature Request]
Hey, 

Thanks for creating this library, it is pretty great! I had a question.

Let's say I have an update server in place, is it possible for this library to check the server every x amount of time for an available update or can I don't something like trigger an update from an MQTT message? I see the `handleUpdate` function, but I am not sure what I would need to set to tell it what to update from. Any help is appreciated!

**Comment by svdrummer** [comment]
Use a timer routine to call the update routines. That was most of us do. Have a look at the ticker library. If you are using deepsleep, count the number of resets.
